src/elements/toniq-button/toniq-button.element.ts

```diff
@@ -19,48 +19,48 @@ export const ToniqButton = defineToniqElement({
     styles: css`
         :host {
             ${toniqFontStyles.boldParagraphFont};
-            display: inline-flex;
-            vertical-align: middle;
             ${noUserSelect};
+            border: 2px solid transparent;
```

Comment:
Moving all these styles into `:host` rather than `button` breaks the focus outline. These styles should remain on `button`:

![broken-button-focus-outline](https://user-images.githubusercontent.com/1205860/178374057-269462b6-0e63-4098-880f-4091d2929371.png)